src/core.js

```diff
@@ -489,10 +489,16 @@ $.extend( $.validator, {
 			this.submitted = {};
 			this.prepareForm();
 			this.hideErrors();
-			var i, elements = this.elements()
+			var elements = this.elements()
 				.removeData( "previousValue" )
 				.removeAttr( "aria-invalid" );
 
+			this.resetElements( elements );
+		},
+
+		resetElements: function( elements ) {
```

Comment:
yeah, after re-reading the source I agree that we used to doc public api and leave the rest in the prototype. so we can leave it as is.